go/libraries/doltcore/sqle/index/dolt_index.go

```diff
@@ -852,7 +852,9 @@ func (di *doltIndex) HasContentHashedField() bool {
 	}
 
 	contentHashedField := false
-
+	if di.indexSch == nil {
```

Comment:
This is probably worth a comment explaining the intent. Seems hard for future code readers to figure out without some more context.